docs/integrations/partner-ecosystem-apps.md

```diff
@@ -22,7 +22,9 @@ Sumo Logic Partner Ecosystem Apps are provided and supported by our partner netw
 * [Cyral](https://github.com/SumoLogic/sumologic-public-partner-apps/tree/master/Cyral)
 * [Dataminr Pulse](https://github.com/SumoLogic/sumologic-public-partner-apps/tree/master/DataminrPulse)
 * [Gigamon HAWK](https://github.com/SumoLogic/sumologic-public-partner-apps/tree/master/Gigamon_HAWK)
+<!-- Heroku? -->
 * [Jamf Protect](https://github.com/SumoLogic/sumologic-public-partner-apps/tree/master/JamfProtect)
+<!-- JFrog? -->
```

Comment:
These comments cause blank lines to appear in the article.

Reply:
thank you so much!! I completely forgot about these bookmarks. just removed.